### PR TITLE
Introduce basic landing page for UI

### PR DESCRIPTION
**Pull request by samuel-gamelin**

### CODE REVIEW

File: src/main/components/MainNav.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Navbar } from 'react-bootstrap';
+import UserElement from '../elements/UserElement';
+
+/**
+ * A component for the main navigation bar.
+ */
+const MainNav = () => (
+  <Navbar bg="dark" variant="dark" sticky="top">
+    <Navbar.Brand href="/" className="mx-auto pl-lg-4">NextCollab</Navbar.Brand>
```
Comment (MohamedRadwan) [inline src/main/components/MainNav.tsx:10]: Following the airbnb standard, you shouldn't use an arrow function here.

File: src/main/components/pages/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import './LandingPage.css';
+
+/**
+ * This component provides the outline for the landing page.
+ */
+const LandingPage = () => (
+  <Container fluid>
+    <Form inline>
+      <FormControl type="text" placeholder="Filter projects" className="mr-sm-2" onKeyPress={(e: React.KeyboardEvent<HTMLInputElement>) => { e.key === 'Enter' && e.preventDefault(); }} />
+    </Form>
```
Comment (MohamedRadwan) [inline src/main/components/pages/LandingPage.tsx:13]: use default function

File: src/main/components/MainNav.tsx (hunk 1 above)
Comment (samuel-gamelin) [reply src/main/components/MainNav.tsx]: Done.

File: src/main/components/pages/LandingPage.tsx (hunk 2 above)
Comment (samuel-gamelin) [reply src/main/components/pages/LandingPage.tsx]: Done.

**Pull request merged**